doc/sphinx/proofs/writing-proofs/rewriting.rst

```diff
@@ -897,3 +811,6 @@ Conversion tactics applied to hypotheses
    definition.
 
    Example: :n:`unfold not in (type of H1) (type of H3)`.
+
+.. tacn:: substitute {? {| -> | <- } } @one_term {? with @bindings }
+   :undocumented:
```

Comment:
You've already inserted this undocumented tactic in the first section, next to `subst`.